test-app/app/templates/index.hbs

```diff
@@ -6,7 +6,8 @@
 >
   <form.field @name='name' as |field|>
```

Comment:
I love this demo <3

Reply:
Not really related, but where does this template get rendered? Because in integration tests, they all seem to render their own templates?

Reply:
Yeah, it was just for seeing something on `htttp://localhost:4200/`, not used in tests at all. With #4 we will probably kill this, and add more feature-complete kitchen-sink like demos there!

Reply:
Thanks for the explanation!